### DISCUSSION TITLE
Using Firebase for Storage

### DISCUSSION THREAD

**Discussion by Trisystema** [shubham0204/FaceRecognition_With_FaceNet_Android#14] [Ideas]
Maybe a stupid question, but would it be possible to use Firebase or another database to save the images instead of storing them locally?

**Comment by shubham0204** [comment]
@Trisystema, We can definitely use Firebase Storage for storing the images. I'll consider this as a feature request for future builds. Instead of the images, you may also store the embeddings ( as a serialized object ).
